search/includes/classes/commands/class-corecommand.php

```diff
@@ -399,7 +401,7 @@ public function delete_index( $args, $assoc_args ) {
 	 * @param array $assoc_args arguments that were passed to the caller command.
 	 * @return void
 	 */
-	public static function confirm_destructive_operation( array $assoc_args ) {
```

Comment:
Why remove the `array` type? The next line tries to access the argument as though it's an array.